### ISSUE TITLE
Inconsistent white background body applied to body

### ISSUE THREAD

**Issue by bcomnes** [beakerbrowser/beaker#817]
Operation System: macOS
Beaker Version: 0.7.10

Beaker is adding a solid white background to bodies, which is inconsisten behavior from other browsers.

![screen shot 2018-01-13 at 12 12 43 pm](https://user-images.githubusercontent.com/166301/34909710-1e6e8b14-f85b-11e7-9ee3-3ff2dba5eba7.png)

Should look like:

![screen shot 2018-01-13 at 12 12 45 pm](https://user-images.githubusercontent.com/166301/34909712-23fdcfae-f85b-11e7-83b0-1d8df14266f4.png)

```
<bret> Bret @pfrazee  @taravancil any clue why https://hypermodul.es/ doesn't display correctly in beaker?
11:56 P<@pfrazee> bret: yeah because we inject "background: #fff" on the body to deal with an issue dealing with webview transparency
11:56 if you open an issue we can address it
```

**Comment by coston** [comment]
I noticed today that this is resolved in v0.8.0.

The issue is at line 689 of app/shell-window/pages.js It says, "// on some devices, if no bg is set, the buffer doesn't get cleared." What does that mean?

**Comment by coston** [comment]
This issue is resolved in v0.8.0, but injecting transparent instead of white works for v0.7.10.

**Comment by pfrazee** [comment]
Fixed
